src/animation/AnimationMixer.js

```diff
@@ -521,8 +521,8 @@ AnimationMixer.prototype = Object.assign( Object.create( EventDispatcher.prototy
 
 		const clipUuid = clipObject !== null ? clipObject.uuid : clip;
 
-		let actionsForClip = this._actionsByClip[ clipUuid ],
-			prototypeAction = null;
+		const actionsForClip = this._actionsByClip[ clipUuid ];
+		let	prototypeAction = null;
```

Comment:
No indentation is needed in such cases. It should be:
```js
const actionsForClip = this._actionsByClip[ clipUuid ];
let prototypeAction = null;
```

Reply:
It's very strange that eslint isn't picking that up. I'll change and commit when I next get some time.

Reply:
Yes, there are many instances of these spaces.
Would be great if you can do a clean up pass.